components/Cta/CtaLink.tsx

```diff
@@ -60,16 +59,12 @@ export const CtaLink = React.forwardRef<HTMLAnchorElement, CtaLinkProps>(
       myLink = url || cachedUrl || href;
     }
 
-    // Add UTM params to internal links
-    //const myLinkWithUtm = useAddUtmParams(myLink);
-    const myLinkWithUtm = myLink;
-
     if (isInternal) {
       return (
         <CtaNextLink
           {...rest}
           ref={ref}
-          href={myLinkWithUtm}
```

Comment:
Don't need em for internal urls

Reply:
Oh so the approach is different from Giving site, but this update makes more sense so we don't get ugly internal URLs 👍🏼